ReactWindows/ReactNative/Storage/FutureAccessList.cs

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Portions derived from React Native:
+// Copyright (c) 2015-present, Facebook, Inc.
```

Comment:
We don't need the `Portions derived` part because this is not derived from Facebook's code.